includes/admin/class-amp-post-meta-box.php

```diff
@@ -338,29 +340,20 @@ public static function get_status_and_errors( $post ) {
 	 * Gets the AMP enabled error message(s).
 	 *
 	 * @since 1.0
-	 * @param string $status The AMP enabled status.
-	 * @param array  $errors The AMP enabled errors.
+	 * @see AMP_Post_Type_Support::get_support_errors()
+	 *
+	 * @param string[] $errors The AMP enabled errors.
 	 * @return array $error_messages The error messages, as an array of strings.
 	 */
-	public function get_error_messages( $status, $errors ) {
+	public function get_error_messages( $errors ) {
```

Comment:
Do you think `AMP_Post_Meta_Box::get_status_and_errors` should be refactored into 2 separate methods? The returned status is now only being used in 1 instance (`AMP_Post_Meta_Box::get_amp_enabled_rest_field`), and the set of errors are used twice elsewhere (`AMP_Post_Meta_Box::enqueue_block_assets` and `AMP_Post_Meta_Box::render_status`).

Reply:
It's being used both in `AMP_Post_Meta_Box::sanitize_status()` and `AMP_Post_Meta_Box::render_status()`, in the latter case it _appears_ to not be used only because the usage is in the `include` file at the end of the method.

Nevertheless, I'm not in love with `get_status_and_errors`.